lib/util/auth.js

```diff
@@ -0,0 +1,105 @@
+import { push } from 'connected-react-router'
+
+/**
+ * Custom links under the user account dropdown.
+ * TODO: Determine/implement the urls below.
+ */
+export const accountLinks = [
+  {
+    text: 'My Account',
+    url: 'account'
+  },
+  {
+    text: 'Help',
+    url: 'help'
+  }
+]
+
+/**
+ * Handle logging out with Auth0 and return to the provided path.
+ */
+export function logout (path = '') {
+  const returnTo = encodeURIComponent(`${getAuthRedirectUri()}${path}`)
+  window.location.href = `https://${process.env.AUTH0_DOMAIN}/v2/logout?returnTo=${returnTo}&client_id=${process.env.AUTH0_CLIENT_ID}`
+}
+
+export const getAuthRedirectUri = () => {
```

Comment:
Is this even needed? I remember running into some issues with admin dashboard and coming up with this function, but I'm not sure if it is relevant here?

Reply:
I am not sure, I took those bits from otp-admin-ui!

Reply:
Ok, so `getAuthRedirectUri()` is needed for Auth0. The `logout()` method above it is not, I'm removing it. Thanks for pointing this out.

Reply:
That's fine, but I don't think`getAuthRedirectUri` is actually needed here either. It's only used one time and I'm not sure if the `return 'http://localhost:9966'` is reachable. Could you try removing and replace the one invocation of this with `${window.location.protocol}//${window.location.host}`?